micrometer-registry-newrelic-telemetry/runtime/src/main/java/io/quarkiverse/micrometer/registry/newrelic/telemetry/NewRelicMeterRegistryProvider.java

```diff
@@ -0,0 +1,73 @@
+package io.quarkiverse.micrometer.registry.newrelic.telemetry;
+
+import static io.micrometer.core.instrument.config.MeterRegistryConfigValidator.check;
+import static io.micrometer.core.instrument.config.MeterRegistryConfigValidator.checkAll;
+
+import java.util.Map;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Singleton;
+
+import com.newrelic.telemetry.micrometer.NewRelicRegistryConfig;
+
+import io.micrometer.core.instrument.config.validate.InvalidReason;
+import io.micrometer.core.instrument.config.validate.Validated;
+import io.micrometer.core.instrument.step.StepRegistryConfig;
+import io.micrometer.core.instrument.util.StringUtils;
+import io.quarkiverse.micrometer.registry.newrelic.telemetry.NewRelicConfig.NewRelicRuntimeConfig;
+import io.quarkiverse.micrometer.registry.newrelic.telemetry.newrelic.NewRelicRegistryConfigAlternative;
+import io.quarkus.arc.DefaultBean;
+import io.quarkus.micrometer.runtime.export.ConfigAdapter;
+
+/**
+ * @see ConditionalRegistryProducer
+ */
+@Singleton
+public class NewRelicMeterRegistryProvider {
+    static final String PREFIX = "newrelic.";
```

Comment:
Note this doesn't have a telemetry segment... 
quarkus.micrometer.export.newrelic.*

That doesn't match what config will provide: 
quarkus.micrometer.export.newrelic.telemetry.*

Is New Relic telemetry expecting "newrelic.telemetry.*" ?

Reply:
fixed